llvm/test/CodeGen/RISCV/GlobalISel/legalizer-info-validation.mir

```diff
@@ -326,16 +326,16 @@
 # DEBUG-NEXT: .. type index coverage check SKIPPED: user-defined predicate detected
 # DEBUG-NEXT: .. imm index coverage check SKIPPED: user-defined predicate detected
 # DEBUG-NEXT: G_SHL (opcode {{[0-9]+}}): 2 type indices, 0 imm indices
-# DEBUG-NEXT: .. type index coverage check SKIPPED: user-defined predicate detected
-# DEBUG-NEXT: .. imm index coverage check SKIPPED: user-defined predicate detected
+# DEBUG-NEXT: .. the first uncovered type index: 2, OK
```

Comment:
How did you fixed the RISCV file? I have to add G_STEP_VECTOR have issues with it.

Reply:
Until yesterday, RISCV didn't call `verify` in RISCVLegalizerInfo.cpp so it couldn't generate the output.